superset-frontend/src/components/Typography/index.tsx

```diff
@@ -16,6 +16,31 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { Typography } from 'antd-v5';
+import { styled, css } from '@superset-ui/core';
+import { Typography as AntdTypography } from 'antd-v5';
 
-export default Typography;
+const StyledLink = styled(AntdTypography.Link)`
+    ${({ theme }) =>
+      css`
+      && {
+        color: ${theme.colorLink};
+        &:hover {
+          color: ${theme.colorLinkHover};
+        }
+    `}
+  }
+`;
+
+export const Typography: typeof AntdTypography & {
+  Text: typeof AntdTypography.Text;
+  Link: typeof StyledLink;
+  Title: typeof AntdTypography.Title;
+  Paragraph: typeof AntdTypography.Paragraph;
+} = Object.assign(AntdTypography, {
+  Text: AntdTypography.Text,
+  Link: StyledLink,
+  Title: AntdTypography.Title,
+  Paragraph: AntdTypography.Paragraph,
+});
```

Comment:
AntdTypography should cover sub-components. Something like this would be enough:
```suggestion
export const Typography: typeof AntdTypography = Object.assign(AntdTypography, {
  Text: AntdTypography.Text,
  Link: StyledLink,
  Title: AntdTypography.Title,
  Paragraph: AntdTypography.Paragraph,
});
```